src/status_im/node/core.cljs

```diff
@@ -134,6 +134,8 @@
               :InstallationID installation-id
               :MaxMessageDeliveryAttempts config/max-message-delivery-attempts
               :MailServerConfirmations  config/mailserver-confirmations-enabled?
+              :VerifyTransactionURL "https://ropsten.infura.io/v3/f315575765b14720b32382a61a89341a" #_"https://mainnet.infura.io/v3/f315575765b14720b32382a61a89341a"
+              :VerifyTransactionChainID 3 #_1
```

Comment:
whats this?

Reply:
we use that to validate transaction, it should always point to mainnet as we don't validate ropsten transactions, so it's not the same as the network the user is in

Reply:
though once we support multiple networks it can probably go away